pipeline/src/node_properties/csi_camera_properties.rs

```diff
@@ -15,6 +15,13 @@ pub struct CsiCameraProperties {
 
 #[derive(Debug, Clone, PartialEq, Serialize, Deserialize)]
 pub struct CsiCameraRuntime {
+    /// Camera URI.
+    ///
+    /// This field is set by API server via camera lookup by `source` field.
+    ///
+    /// URI is used by `lumeod` to access local camera.
```

Comment:
* nit: not sure this comment provides any new info to reader.
* "This URI is..".

Reply:
It just gives a hint why this field was introduced and with the above one we have two clarifications: 
- who sets this field
- who uses this field